app/graphql/types/rock_and_pebble_type.rb

```diff
@@ -1,20 +1,20 @@
 module Types
-    class RockAndPebbleType < Types::BaseObject
-        field :id, ID, null: false
-        field :rock, Types::UserType, null: false
-        field :pebble, Types::UserType, null: false
-        field :active, Boolean, null: false
-        field :my_rocks, [Types::UserType], null: true
-        field :my_pebbles, [Types::UserType], null: true
+  class RockAndPebbleType < Types::BaseObject
+      field :id, ID, null: false
+      field :rock, Types::UserType, null: true
+      field :pebble, Types::UserType, null: true
+      field :active, Boolean, null: false
+      field :my_rocks, [Types::UserType], null: true
+      field :my_pebbles, [Types::UserType], null: true
 
-        def my_rocks
-          user = User.find(object.pebble_id)
-          user.rocks
-        end
+  def my_rocks
+    user = User.find(object.pebble_id)
+    user.rocks
+  end
 
-        def my_pebbles
-          user = User.find(object.rock_id)
-          user.pebbles
-        end
-    end
+  def my_pebbles
+    user = User.find(object.rock_id)
+    user.pebbles
+  end
+  end
```

Comment:
I'm not positive b/c of the GitHub formatting but there might be an extra `end` here.

Reply:
Thanks! Just fixed the indentation for readability here.